### PR TITLE
[Sprint Maintain]Feature Create UI Statistic Of Personal Next

### PR DESCRIPTION
**Pull request by nguyenhuy95dn**
Feature Create UI Statistic Of Personal
https://dev.framgia.com/redmine/issues/131118
Effect: N/A

### CODE REVIEW

File: app/src/main/java/com/framgia/wsm/screen/statistics/StatisticContainerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            default:
+                return null;
+        }
+    }
+
+    public Fragment getFragment(@SearchResultsTab int position) {
+        return mFragments.get(position);
+    }
+
+    @IntDef({ SearchResultsTab.TAB_MONTH, SearchResultsTab.TAB_YEAR })
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/statistics/StatisticContainerAdapter.java]: WARNING: Missing a Javadoc comment.

File: app/src/main/java/com/framgia/wsm/screen/statistics/StatisticContainerAdapter.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    public @interface SearchResultsTab {
+        int TAB_MONTH = 0;
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/statistics/StatisticContainerAdapter.java]: ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.

File: app/src/main/java/com/framgia/wsm/screen/statistics/StatisticContainerAdapter.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        int TAB_YEAR = 1;
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/statistics/StatisticContainerAdapter.java]: ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -317,5 +317,6 @@
     <string name="request_leave_legend">IL/LE/LO</string>
     <string name="request_ot_approved">Request Ots approved</string>
     <string name="there_is_no_request_to_choose">There is no request to choose</string>
+    <string name="year">year</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`year`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/java/com/framgia/wsm/screen/statistics/StatisticContainerAdapter.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+import com.framgia.wsm.screen.holidaycalendar.HolidayCalendarFragment;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by nguyenhuy95dn on 8/3/2017.
+ */
+
+public class StatisticContainerAdapter extends FragmentPagerAdapter {
+    private static final int TAB_NUMBER = 2;
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/statistics/StatisticContainerAdapter.java]: rename to ``TOTAL_TAB``

**Pull request merged**